mtenn/tests/test_model_config.py

```diff
@@ -65,3 +66,25 @@ def test_random_seed_schnet():
         for p1, p2 in zip(set_model1.parameters(), set_model2.parameters())
     ]
     assert sum(set_equal) == len(set_equal)
+
+@pytest.mark.skipif(not HAS_VISNET, reason="requires VisNet from nightly PyG")
+def test_random_seed_visnet():
```

Comment:
another test to add would be one that creates a `mtenn.conversion_utils.ViSNet` object from an existing `from torch_geometric.nn.models.ViSNet` object. I'm realizing now that this isn't actually supported with the `Config` method of model construction, but can still be done directly with the `conversion_utils.ViSNet` class.